test/Microsoft.AspNetCore.Routing.Tests/Template/TemplateBinderTests.cs

```diff
@@ -720,7 +720,7 @@ public void TemplateBinder_KeepsExplicitlySuppliedRouteValues_OnFailedRouetMatch
             var template = "{area?}/{controller=Home}/{action=Index}/{id?}";
             var encoder = new UrlTestEncoder();
```

Comment:
Don't use `UrlTestEncoder` anywhere except tests specifically about unusual override scenarios or encoding tests (that aren't looking for `%xx`).